db_assessment/import_db_assessment.py

```diff
@@ -553,6 +573,14 @@ def getObjNameFromFiles(fileName,splitterChar,pos):
     
     return None    
 
+def getPkeyFromFile(fileName):
```

Comment:
Consider using:

tableName = getObjNameFromFiles(fileName,'__',1)

Reply:
getPkeyFromFile  is removed in latest commit for bug132 branch